### ISSUE TITLE
Use specific lodash dependencies to avoid pulling in entire library

### ISSUE THREAD

**Issue by ajsharp** [cape-io/lodash-humps#8]

**Comment by webmasterkai** [comment]
I thought the https://github.com/lodash/babel-plugin-lodash babel lodash plugin took care of this automatically. Is it not working as expected? Maybe I have a bad config somewhere?

**Comment by Oleksii14** [comment]
@webmasterkai yeah, right. But on some platforms, e.g. Vite js, this plugin does not work. Can you please change the imports to be like

```
import camelCase from 'lodash/camelCase'
```
?
Thanks in advance 👍🏻
